services/QuillLMS/client/app/bundles/Grammar/actions/fillInBlankQuestions.ts

```diff
@@ -3,6 +3,7 @@ import _ from 'underscore';
 import { ActionTypes } from './actionTypes';
 import { populateQuestions, setSessionReducerToSavedSession } from './session.ts';
 import { startListeningToActivities } from './grammarActivities';
+import { getQuestion } from './questions';
```

Comment:
The only changes I see in this file are this import and the new `submitEditedQuestion`, but that function doesn't seem to use `getQuestion`.  So is the import actually being used?